Duplicati/Library/Main/Operation/Backup/BackendUploader.cs

```diff
@@ -86,18 +115,18 @@ public static Task Run(Common.BackendHandler backend, Options options, Common.Da
             async self =>
             {
                 var inProgress = new Queue<KeyValuePair<int, Task>>();
-                var max_pending = options.AsynchronousUploadLimit == 0 ? long.MaxValue : options.AsynchronousUploadLimit;
-                var noIndexFiles = options.IndexfilePolicy == Options.IndexFileStrategy.None;
+                var max_pending = m_options.AsynchronousUploadLimit == 0 ? long.MaxValue : m_options.AsynchronousUploadLimit;
+                var noIndexFiles = m_options.IndexfilePolicy == Options.IndexFileStrategy.None;
                 var active = 0;
                 var lastSize = -1L;
                 
-                while(!await self.Input.IsRetiredAsync && await taskreader.ProgressAsync)
+                while(!await self.Input.IsRetiredAsync && await m_taskreader.ProgressAsync)
                 {
                     try
                     {
                         var req = await self.Input.ReadAsync();
 
-                        if (!await taskreader.ProgressAsync)
+                        if (!await m_taskreader.ProgressAsync)

```

Comment:
I am not a fan of the m_* member variable "convention" and would like to phase them out. I know the code is written mostly in such a way that it's not clear what local vars and member vars are -mainly due to large functions. It's quite possible I am the only one disliking the m_ style notation in duplicati team.

Reply:
I dislike the m_* style as well. Being my first PR for the project I didn't want to step on any toes and kept with the current style. If there is an agreed upon standard I would be happy to switch to using that. I'm partial to m* or to no prefix at all myself.

Reply:
If it were up to me, I would prefer to always use the `this` keyword when referring to instance members.  At least for me, it is an immediate and clear indication that the variable refers to an instance member.

I have also tried to refrain from disturbing the local style too much, as in the end, consistency should perhaps be the ultimate goal.

Reply:
warwickmm that's actually the other one I dislike. :) But I can get used to anything as long as it's consistent. If I really need to know the scope of a variable Visual Studio will show that in the Intellisense tooltip. I realize not everyone has that information available to them though. And I generally like to keep functions short enough that it is easy to tell anyway.

Reply:
For me, moving the cursor and hovering is too much effort (especially on a laptop).  But yes, as long as we're consistent I am fine with any style.

Reply:
I looked through a lot of files and most of them are using the m_ style. I'll plan on leaving this as-is for now. I think having a coding standard should be a separate conversation where all interested parties can help make the final decision.
If the community agrees on a different standard before this PR is merged I can always update it then.

Reply:
Sounds good.  Thanks.

Reply:
That's fine.  I just noticed that you changed it to the m_* style actively, whence the remark. We should discuss coding styles in another place and you shouldn't change it here. I hope I can find some time soon to read the actual code of your implemented interesting feature.